openmetadata-ui/src/main/resources/ui/src/utils/CommonUtils.tsx

```diff
@@ -1040,3 +1043,20 @@ export const getIngestionFrequency = (pipelineType: PipelineType) => {
       return getDayCron(value);
   }
 };
+
+//  return the ingestion state
+export const getIngestionStatusState = (
+  current: IngestionCurrentState,
+  ingestion: IngestionPipeline,
+  text: string
+) => {
+  return current.id === ingestion.id ? (
```

Comment:
use generic name instead of `getIngestionStatusState`, same for `ingestion` we only need ingestion id and not full object and make name accordingly